packages/flutter/test/painting/border_radius_test.dart

```diff
@@ -187,8 +187,8 @@ void main() {
     expect(borderRadius.resolve(TextDirection.ltr).toRRect(rect), RRect.fromRectXY(rect, 3.0, 3.0));
     expect(borderRadius.resolve(TextDirection.rtl).toRRect(rect), RRect.fromRectXY(rect, 3.0, 3.0));
 
-    const Radius radius1 = Radius.elliptical(89.0, 87.0);
-    const Radius radius2 = Radius.elliptical(103.0, 107.0);
+    const Radius radius1 = Radius.elliptical(8.9, 8.7);
+    const Radius radius2 = Radius.elliptical(9.3, 9.7);
```

Comment:
We should also add tests that assert a radius larger than the rect gets clamped.

I can help you running this through the Google monorepo as well.

Reply:
Yes, I need to make a suite of tests. I just wanted to see if it would break something first.

Reply:
Added them.